packages/core/src/lib/features/FeatureManager.ts

```diff
@@ -43,11 +44,19 @@ export class FeatureManager {
     variables: new Set<string>(),
     launchUrl: new Array<string>(),
   };
+  delegationService = {
+    imports: new Set<string>(),
+    constructor: new Array<string>(),
+  };
 
   /**
    * Builds a new intance from a TwaManifest.
    */
   constructor(twaManifest: TwaManifest) {
+    if (twaManifest.features.locationDelegation) {
```

Comment:
Can this be `twaManifest.features.locationDelegation && twaManifest.features.locationDelegation.enabled`?